examples/angular-example/tsconfig.app.json

```
@@ -0,0 +1,10 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
```

Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this (does the functionality still work in VS Code?):

```suggestion
{
  "_to_learn_more_about_this_file_see": "https://angular.io/config/tsconfig",
```